From 985f2a009ddf5179482e7551400cf8065553707e Mon Sep 17 00:00:00 2001
From: Michael Catanzaro <mcatanzaro@igalia.com>
Date: Wed, 11 Apr 2018 22:16:54 -0500
Subject: Revert "Revert "Don't hard-enforce strong passwords""

This reverts commit 0c67a9800c7832dd55c046adc372833a96a96cf0.

This is a requirement for Fedora, but I suspect few distros want to
irritate users right off the bat by dictating which passwords may be
used.
---
 gnome-initial-setup/pages/password/gis-password-page.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gnome-initial-setup/pages/password/gis-password-page.c b/gnome-initial-setup/pages/password/gis-password-page.c
index cf3db9e..731666d 100644
--- a/gnome-initial-setup/pages/password/gis-password-page.c
+++ b/gnome-initial-setup/pages/password/gis-password-page.c
@@ -59,7 +59,7 @@ page_validate (GisPasswordPage *page)
 {
   GisPasswordPagePrivate *priv = gis_password_page_get_instance_private (page);
 
-  return priv->valid_confirm && priv->valid_password;
+  return priv->valid_confirm;
 }
 
 static void
-- 
cgit v0.12

